modules/setting/attachment.go

```diff
@@ -29,7 +29,7 @@ func newAttachmentService() {
 
 	Attachment.Storage = getStorage("attachments", storageType, sec)
 
-	Attachment.AllowedTypes = sec.Key("ALLOWED_TYPES").MustString(".docx,.gif,.gz,.jpeg,.jpg,.log,.pdf,.png,.pptx,.txt,.xlsx,.zip")
+	Attachment.AllowedTypes = sec.Key("ALLOWED_TYPES").MustString(".docx,.gif,.gz,.jpeg,.jpg,.mp4,.log,.pdf,.png,.pptx,.txt,.xlsx,.zip")
```

Comment:
I know it is kinda unrelated, but why are `svg`, `md`, and `csv` missing?
Those three are also fairly popular formats with basically no security implications.

Reply:
> `md`, and `csv`

Both looks good to me to be added...

> `svg`

I haven't looked all that deep into the code that serves the actual file, but `svg` allows for inline script(and we don't want to serve malicious files) also https://github.com/golang/go/issues/15888

Reply:
^ Related to svg #1095

Reply:
Okay, I didn't know that you can embed JS inside `svg`s.
I mean, it makes sense that it depends on the renderer whether it's actually interpreted or not.

On the other hand, we also allow `docx` files that can contain macros.

Reply:
> I mean, it makes sense that it depends on the renderer whether it's actually interpreted or not.

Yeah, if I know that it will be downloaded and not executed, then we can include it safely.

> On the other hand, we also allow `docx` files that can contain macros.

Yeah, but docx aren't executed immediately within the browsers. And nowadays the popular software open most `docx` in a safe mode.